### PR TITLE
[JN-1375] UX tweaks for kit scanning

### PR DESCRIPTION
**Pull request by MatthewBemis**
#### DESCRIPTION (include screenshots, and mobile screenshots for participant UX)

This PR handles the lowest hanging fruit, which were the minor admin UI-only tweaks.

A second PR will have some backend changes that improve error messaging.

Fixes these points of feedback:

* DOB and Full Name fields shouldn’t look like input fields
* More notes about permissions requirements for allowing camera use
* When enrollee QR code is invalid, don’t display an error toast on top of the “Did you scan QR the correct QR code?”
* Add note to the manual shortcode override info popup that we only use the alphabet for shortcodes (so no need to worry about swapping an O and a 0)
* Add a spinner on submit
* Make sure long tracking numbers are fully visible (the tail end can get cut off on smaller screens)
* When opening the barcode scanner, sometimes it’s partially off-screen. And sometimes it finishes scanning before the admin can even see what’s going on- which can be jarring/confusing
* Automatically collapse sidebar when loading the kit scanner (and maybe change expand/collapse icon to a caret so it doesn’t look as much like an “exit” icon) 

#### TO TEST:  *(simple manual steps for confirming core behavior -- used for pre-release checks)*

Go to the admin kit scanning tool and verify all of the above points were addressed

### CODE REVIEW

File: ui-admin/src/navbar/AdminSidebar.tsx [hunk 1]
```diff
@@ -44,6 +44,13 @@ const AdminSidebar = ({ config }: { config: Config }) => {
 
   const color = ZONE_COLORS[config.deploymentZone] || ZONE_COLORS['prod']
 
+  //if current path is /kits/scan, we automatically collapse the sidebar
+  useEffect(() => {
+    if (window.location.pathname.endsWith('kits/scan')) {
```
Comment (devonbush) [inline ui-admin/src/navbar/AdminSidebar.tsx]: maybe pull this out into a helper function "isMobileFirstRoute"

File: ui-admin/src/study/kits/kitcollection/KitScanner.tsx [hunk 2]
```diff
@@ -162,39 +170,46 @@ export const KitScanner = ({ studyEnvContext }: { studyEnvContext: StudyEnvConte
...
         Click the button below to open the camera and scan the enrollee&apos;s unique QR code.
         <InfoPopup content={
-          <>The enrollee can find their unique QR code by going to the kits page on their profile.
-          If you are unable to use your camera or scan the barcode for any reason, you may override
-          manually and enter the participant code (shortcode) found directly under their QR code.</>}/>
+          <>
+            The enrollee can find their unique QR code by going to the kits page on their profile.
+            If you are unable to use your camera or scan the barcode for any reason, you may override
+            manually and enter the participant code (shortcode) found directly under their QR code. Note
+            that participant shortcodes only contain letters, and will <b>not</b> contain any numbers.
```
Comment (devonbush) [inline ui-admin/src/study/kits/kitcollection/KitScanner.tsx]: wrap the "Note" in `<span className="text-muted fst-italic">` to break up the text a bit more

File: ui-admin/src/study/kits/kitcollection/KitScanner.tsx [hunk 3]
```diff
@@ -217,19 +232,23 @@ export const KitScanner = ({ studyEnvContext }: { studyEnvContext: StudyEnvConte
...
-      <TextInput
-        label={'Date of Birth'}
-        className="mb-3" disabled={true}
-        value={enrollee ? dateToDefaultString(enrollee.profile.birthDate) : ''}>
-      </TextInput>
+      {enrollee ? <>
+        <label className={'fw-bold'}>Full Name</label>
+        <div className="mb-3">
+          {enrollee ?
+            `${enrollee.profile.givenName} ${enrollee.profile.familyName}` : 'Participant has not provided a name'}
```
Comment (devonbush) [inline ui-admin/src/study/kits/kitcollection/KitScanner.tsx]:
```suggestion
            `${enrollee.profile.givenName} ${enrollee.profile.familyName}` : 'not yet entered'}
```

File: ui-admin/src/study/kits/kitcollection/KitScanner.tsx [hunk 4]
```diff
@@ -217,19 +232,23 @@ export const KitScanner = ({ studyEnvContext }: { studyEnvContext: StudyEnvConte
...
+        </div>
+        <label className={'fw-bold'}>Date of Birth</label>
+        <div className="mb-3">
+          {enrollee.profile.birthDate ?
+            dateToDefaultString(enrollee.profile.birthDate) : 'Participant has not provided a date of birth'}
```
Comment (devonbush) [inline ui-admin/src/study/kits/kitcollection/KitScanner.tsx]:
```suggestion
            dateToDefaultString(enrollee.profile.birthDate) : 'not yet entered'}
```

**Review by devonbush — APPROVED** [review]
Nice, this packs in a lot of improvements.  Some suggestions for making the UX a bit more concise

**Review by connorlbark — APPROVED** [review]

**Pull request merged**